README.md

```diff
@@ -117,13 +160,16 @@ To workaround that, we allow you to provide additional information on your Graph
 
 To provide that, we are using `@graphql-tools` loaders to load your sibling operations and fragments, just specify a glob expression(s) that points to your code/.graphql files:
 
-```json
+```diff
 {
   "files": ["*.graphql"],
   "parser": "@graphql-eslint/eslint-plugin",
   "plugins": ["@graphql-eslint"],
+  "rules": {
+    "unique-operation-name": ["error"]
```

Comment:
must be `@graphql-eslint/unique-operation-name`, btw don't worry I'll fix this

Reply:
Good catch! The same probably needs added to line 141.